app/res/values/strings.xml

```diff
@@ -450,4 +452,144 @@
     <string name="dependency_missing_dialog_message_plural" cc:translatable="true">CommCare needs below applications to be installed on your device before you can continue-</string>
     <string name="dependency_missing_dialog_go_to_store" cc:translatable="true">Install from Store</string>
     <string name="dependency_missing_dialog_playstore_not_found" cc:translatable="true">No Play Store found on your device</string>
+
+    <string name="app_select_message" cc:translatable="true">Select an app:</string>
+
+    <!-- ConnectID strings -->
+
+    <string name="connect_id_enabled" cc:translatable="true">ConnectID Enabled</string>
+    <string name="login_button_connect" cc:translatable="true">Unlock ConnectID</string>
+    <string name="login_welcome_connect_signed_out" cc:translatable="true">Unlock ConnectID first and then enter your app.</string>
+    <string name="login_welcome_connect_signed_in" cc:translatable="true">Welcome %s!</string>
+
+    <string name="choice_or" cc:translatable="true">_____________    OR    _____________</string>
+    <string name="busy_message" cc:translatable="true">Still busy…</string>
```

Comment:
Maybe give a more descriptive message - `We are still busy processing your previous request. Please try again in some time`